src/style/apply.js

```diff
@@ -206,6 +206,13 @@ styfn.applyContextStyle = function( cxtMeta, cxtStyle, ele ){
     // save cycles when the context prop doesn't need to be applied
     if( eleProp === cxtProp ){ continue; }
 
+    // save cycles when a mapped context prop doesn't need to be applied
+    if( is.fn( cxtProp.value )) {
```

Comment:
Generally if `cxtProp.mapped != null`

The function case is probably cheaper as [`cxtProp.mapped === types.fn`](https://github.com/cytoscape/cytoscape.js/blob/unstable/src/style/parse.js#L99)

Reply:
Makes sense!